`opencloud posixfs scan` command aborting with nil derefence

`opencloud posixfs scan` give this backtrace when it looses the connection to NATS:

```
# podman exec opencloud opencloud posixfs scan
Starting posixfs scan...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x55cc663a514e]

goroutine 101 [running]:
github.com/rs/zerolog.(*Logger).disabled(...)
        github.com/rs/zerolog@v1.35.0/log.go:513
github.com/rs/zerolog.(*Logger).should(0xc0010b5f7c?, 0xf8?)
        github.com/rs/zerolog@v1.35.0/log.go:518 +0xe
github.com/rs/zerolog.(*Logger).newEvent(0x0, 0x3, 0x0)
        github.com/rs/zerolog@v1.35.0/log.go:489 +0x36
github.com/rs/zerolog.(*Logger).Error(...)
        github.com/rs/zerolog@v1.35.0/log.go:370
github.com/opencloud-eu/reva/v2/pkg/storage/cache.NewNatsKeyValue.func1(0x10000?, {0x55cc6a9ed918, 0x55cc6c897300})
        github.com/opencloud-eu/reva/v2@v2.46.7/pkg/storage/cache/kv.go:30 +0x28
github.com/nats-io/nats%2ego.(*Conn).doReconnect.func1()
        github.com/nats-io/nats.go@v1.51.0/nats.go:3167 +0x23
github.com/nats-io/nats%2ego.(*asyncCallbacksHandler).asyncCBDispatcher(0xc00109bb40)
        github.com/nats-io/nats.go@v1.51.0/nats.go:3485 +0x1e
created by github.com/nats-io/nats%2ego.Options.Connect in goroutine 1
        github.com/nats-io/nats.go@v1.51.0/nats.go:1864 +0x385
Error: timed out waiting for file /var/lib/containers/storage/overlay-containers/5a6f46035fd9b61cdc11471b1fc16ca1a9ae3c021274126fe1bef258de5b456f/userdata/d4959940ee082580771da056d0ab9e82190097c62adfd46b72ac82165a79e24d/exit/5a6f46035fd9b61cdc11471b1fc16ca1a9ae3c021274126fe1bef258de5b456f
```

--- a/opencloud/pkg/command/posixfs.go
+++ b/opencloud/pkg/command/posixfs.go
@@ -12,6 +12,7 @@ import (
 	"github.com/opencloud-eu/opencloud/pkg/config"
 	"github.com/opencloud-eu/opencloud/pkg/config/configlog"
 	"github.com/opencloud-eu/opencloud/pkg/config/parser"
+	oclog "github.com/opencloud-eu/opencloud/pkg/log"
 	storageUsersParser "github.com/opencloud-eu/opencloud/services/storage-users/pkg/config/parser"
 	"github.com/opencloud-eu/opencloud/services/storage-users/pkg/event"
 	"github.com/opencloud-eu/opencloud/services/storage-users/pkg/revaconfig"
@@ -105,6 +106,11 @@ func scanCmd(ocCfg *config.Config) *cobra.Command {
 				fmt.Fprintf(os.Stderr, "Failed to create event stream for posix driver: %v\n", err)
 				os.Exit(1)
 			}
+			log := oclog.NewLogger(
+				oclog.Name("posixfs scan"),
+				oclog.Level("error"),
+				oclog.Pretty(true),
+				oclog.Color(false)).Logger
 
 			f, ok := registry.NewFuncs["posix"]
 			if !ok {
@@ -112,7 +118,7 @@ func scanCmd(ocCfg *config.Config) *cobra.Command {
 				os.Exit(1)
 			}
 
-			fs, err := f(drivers["posix"].(map[string]any), fsStream, nil)
+			fs, err := f(drivers["posix"].(map[string]any), fsStream, &log)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Failed to initialize filesystem driver '%s': %v\n", cfg.Driver, err)
 				return err

